lua/tardis/screens/cl_visualgui_settings.lua

```diff
@@ -0,0 +1,55 @@
+TARDIS:AddSetting({
+	id="visual_gui_enabled",
+	name="Visual GUI Enabled",
+	desc="Should new visual GUI be used?",
+	section="Misc",
+	value=false,
+	type="bool",
+	option=true,
+	networked=false
+})
+
+TARDIS:AddSetting({
+	id="visual_gui_bigpopup",
+	name="Visual GUI big popup",
+	desc="Should the popup for new visual GUI be bigger?",
+	section="Misc",
+	value=true,
+	type="bool",
+	option=true,
+	networked=false
+})
+
+TARDIS:AddSetting({
+	id="visual_gui_screen_numrows",
+	name="Visual GUI rows (screen)",
+	desc="How many rows of buttons should the visual GUI on the screen have?",
+	section="Misc",
+	type="number",
+	min=2,
+	max=7,
+	option=true,
+	value=3
```

Comment:
I think 4 is a good number by default instead of 3

Reply:
I'd leave it to 3 now, because:
- There are currently small screens like 2005 also working, and 4 rows are just impossible for them
- I will be adding per-interior default values for numbers of rows in the further updates and an option to override them, then the default can be 4
- Tbh, I really like the way three rows align in the default/capaldi screen resolution. It takes 2 scrolls exactly for all controls with no overlapping and no free space left (just a perfectionist screaming inside of me, lol)

Reply:
Okay, yes I think the interior file default setting is a good idea for this and the option in the settings can be more of an override perhaps or maybe scale relative to the interior setting, so I am fine for this to remain as is for now